config/manifests/bases/multicluster-gateway-controller.clusterserviceversion.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: multicluster-gateway-controller.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: test
```

Comment:
should these change?

Reply:
hmmm yes it should, let me see why that's not updating